### DISCUSSION TITLE
I have problem sending audio to iPhone.

### DISCUSSION THREAD

**Discussion by sg-sg-sg-sg-sg-sg** [WhiskeySockets/Baileys#22] [General]
I convert the audio to .opus and it was working fine.
Then these days it stopped working. Does anyone have an idea how to fix it? On Android it's working normally.

obs: The .ogg files don't work either. use ffmpeg to convert.

**Comment by andresayac** [comment]
![image](https://user-images.githubusercontent.com/16548795/234073026-8ec4bde2-4ef0-4810-bde6-dcfd06ced64f.png)
https://github.com/codigoencasa/bot-whatsapp/blob/main/packages/provider/src/utils/convertAudio.js#L10
https://github.com/codigoencasa/bot-whatsapp/blob/c9cdd393a0533282438ece011e2e0195717b56d0/packages/provider/src/baileys/index.js#L198

**Reply by sg-sg-sg-sg-sg-sg** [reply]
It didn't work, like it's more or less like this.

opus - works on android
mp3 - works on iphone

but not on the contrary

**Comment by textarea** [comment]
Use .mp4

**Reply by sg-sg-sg-sg-sg-sg** [reply]
use it as video or audio even?

**Reply by textarea** [reply]
Yes, for both

**Reply by sg-sg-sg-sg-sg-sg** [reply]
did not work

**Reply by textarea** [reply]
what happens? doesn't it play?

**Reply by sg-sg-sg-sg-sg-sg** [reply]
see if that's the case.

take a .mp3 file convert to mp4

send with

{ audio: { url: files , mimetype: 'audio/mp4', ppt: true } }

if that's the case, don't touch

**Reply by textarea** [reply]
try changing: { audio: { url: file } , mimetype: 'audio/mp4', ppt: true }

**Reply by sg-sg-sg-sg-sg-sg** [reply]
neither, executes but stays silent

**Reply by sg-sg-sg-sg-sg-sg** [reply]
you can do it normally?

**Reply by textarea** [reply]
yes i send normally audio message to iOS and Android devices, I had troubles too at the beginning when I wa trying .mp3, .ogg, .wav files... then I reached with .mp4 files

**Reply by sg-sg-sg-sg-sg-sg** [reply]
I did some tests here, but one will not.

Do you use ffmepg to convert?

**Reply by textarea** [reply]
yes
`ffmpeg(tempName)
          .output(`${tempName}.mp4`)
          .toFormat('mp4')
          .on('end', () => {`

**Reply by sg-sg-sg-sg-sg-sg** [reply]
I tested it that way, and it's still the same. It works another time it doesn't work
